<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMS_Dialog</class>
 <widget class="QDialog" name="IMS_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>517</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>IMS_Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/res/head.jpg</normaloff>:/res/head.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>321</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="page_friend">
    <attribute name="title">
     <string>好友</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="page_tempfriend">
    <attribute name="title">
     <string>新朋友</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>371</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>319</width>
        <height>369</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="wdg_list" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>340</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_group">
    <attribute name="title">
     <string>群聊</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_tool">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>490</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>201</width>
     <height>69</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lb_name">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>刘来乐</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_feeling">
      <property name="font">
       <font>
        <family>新宋体</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>啊风蚀高地</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
